components/card/card.html

```diff
@@ -0,0 +1,156 @@
+                    <div class="card-content-h display-flex-c">
+                        <h3 class="card-title heading-3">Denim Jacket</h3>
+                        <small class="card-author">by Studium</small>
+                        <p class="card-text heading-4">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, unde!
+                        </p>
+                        <div class="card-footer-h">
+                            <button class="card-btn btn btn-solid-primary">Add to Cart</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            
+            <!-- card with text-overlay -->
+            <h4 class="heading-4">Card with text overlay</h4>
+            <div class="card display-flex-c pos-rltv">
+                <div class="card-img">
+                    <img src="../../assets/card-2.jpg" alt="Air pods">
+                </div>
+                <div class="card-content display-flex-c">
+                    <h3 class="card-title heading-3">Air pods</h3>
+                    <small class="card-author">by Studium</small>
+                    <p class="card-text heading-4">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, unde!
+                    </p>
+                    <div class="card-footer">
+                        <button class="card-btn btn btn-solid-primary">Add to Cart</button>
+                    </div>
+                    <div class="text-overlay">
+                        <h3 class="text-overlay-content heading-3">Out of Stock</h3>
+                    </div>
+                </div>
+            </div>
+
+            <!-- text only card -->
+            <h4 class="heading-4">Text only card</h4>
+            <div class="card">
+                <div class="card-content p-1">
+                    <h3 class="card-title heading-3">Air pods</h3>
+                    <small class="card-author">by Studium</small>
+                    <p class="card-text heading-4">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, unde!
+                    </p>
+                </div>
+            </div>
+
+            <!-- card with badge -->
+            <h4 class="heading-4">Card with badge</h4>
+            <!-- <div class="card display-flex-c">
+                <div class="card-img">
+                    <img src="../../assets/card-2.jpg" alt="Air pods">
+                </div>
+                <div class="card-content display-flex-c">
+                    <h3 class="card-title heading-3">Air pods</h3>
+                    <small class="card-author">by Studium</small>
+                    <p class="card-text heading-4">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, unde!
+                    </p>
+                    <div class="card-footer">
+                        <button class="card-btn btn btn-solid-primary">Add to Cart</button>
+                    </div>
+                </div>
+            </div> -->
+
+
```

Comment:
if this is not needed then you should remove this comment

Reply:
I am making this one as card with badge